cmd/config.go

```diff
@@ -5,17 +5,26 @@ import (
 	"io/ioutil"
 	"strings"
 
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+
 	"gopkg.in/yaml.v2"
 )
 
+type ResourceConfig struct {
+	Targets  types.Set `yaml:"targets"`
+	Excludes types.Set `yaml:"excludes"`
```

Comment:
Shouldn't this be `yaml:"target"` and `yaml:"exclude"` ?

Reply:
IMO not, since you specify a list.

Reply:
Makes sense. Doesn't this need to be `targets`: https://github.com/rebuy-de/aws-nuke/pull/74/files#diff-04c6e90faac2675aa89e2176d2eec7d8R223 ?

Reply:
fexid